pyproject.toml

```diff
@@ -34,7 +34,6 @@ include = []
 exclude = [".github/", "tests/"]
 
 [tool.isort]
-skip = ["venv"]
```

Comment:
It seems isort behavior has changed regarding this conf. Previously it was helpful, now it just overrides the default skip, and doesn't even support folders (there's `skip_glob` I think for that).